internal/super/flix.go

```diff
@@ -234,9 +316,56 @@ func getTemplate(state *flowkit.State, flixService flixkit.FlixService, flixQuer
 		if err != nil {
 			return nil, fmt.Errorf("could not parse flix from file %s: %w", flixQuery, err)
 		}
+	case flixUrl:
+		flixString, err := flixkit.FetchFlixWithContext(ctx, flixQuery)
+		if err != nil {
+			return nil, fmt.Errorf("could not parse flix from url %s: %w", flixQuery, err)
+		}
+		template, err = flixkit.ParseFlix(flixString)
+		if err != nil {
+			return nil, fmt.Errorf("could not parse flix from url %s: %w", flixQuery, err)
+		}
 
 	default:
 		return nil, fmt.Errorf("invalid flix query type: %s", flixQuery)
 	}
 	return template, nil
 }
+
+// GetRelativePath computes the relative path from target to source.
+func GetRelativePath(source, target string) (string, error) {
+	relPath, err := filepath.Rel(filepath.Dir(target), source)
+	if err != nil {
+		return "", err
+	}
+
+	// If the file is in the same directory and doesn't start with "./", prepend it.
+	if !filepath.IsAbs(relPath) && relPath[0] != '.' {
+		relPath = "./" + relPath
+	}
+
+	return relPath, nil
+}
+
+func GetDeployedContracts(state *flowkit.State) []flixkit.Contracts {
```

Comment:
This function will only get contracts in the `deployments` section of flow.json.  Do you need aliased contracts as well?

i.e.
```
{
"contracts": {
    "Flowvatar": {
        aliases: {
            testnet: "0x1234"
        }
    }
}
```

Reply:
I was only thinking of user's deployed contracts. Good call I can add contract aliases.

Reply:
You know what the flow-cli command is to add an alias?

Reply:
Update PR to add alias contracts.